manifests/r/RStudio/quarto/1.1.165/RStudio.quarto.locale.en-US.yaml

```diff
@@ -0,0 +1,40 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RStudio.quarto
+PackageVersion: 1.1.165
+PackageLocale: en-US
+Publisher: Rstudio, PBC
+PublisherUrl: https://www.rstudio.com/
+PackageName: Quarto
+PackageUrl: https://quarto.org/
+License: GPL v2
+LicenseUrl: https://quarto.org/license.html
+Copyright: Copyright (C) RStudio, PBC
+CopyrightUrl: https://quarto.org/trademark.html
+ShortDescription: Quarto® is an open-source scientific and technical publishing system built on Pandoc
+Description: |-
+  Main features of Quarto:
+     Create dynamic content with Python, R, Julia, and Observable.
+     Author documents as plain text markdown or Jupyter notebooks.
+     Publish high-quality articles, reports, presentations, websites, blogs, and books in HTML, PDF, MS Word, ePub, and more.
+     Author with scientific markdown, including equations, citations, crossrefs, figure panels, callouts, advanced layout, and more.
+Moniker: quarto
+Tags:
+- document
+- jupyter
+- markdown
+- ojs
+- pandoc
+- publishing
+- r
+- reproducibility
+- rmarkdown
+Documentation:
+  - DocumentLabel: Tutorial
+    DocumentUrl: https://quarto.org/docs/get-started/
+  - DocumentLabel: Documentation
+    DocumentUrl: https://quarto.org/docs/guide/

```

Comment:
Just curious: what is this used for ? Isn't `PackageUrl` enough to find the Quarto website ?

Reply:
Yes, but by that measure we could also omit `LicenseUrl` and `CopyrightUrl`, couldn't we? 😉 
It is in the [schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.2.0/locale.md) in any case (though not yet fully supported AFAICT).

Reply:
This is just useful data which the client or third party clients can use. The more info the better.

Reply:
Ok. Licence and Copyright have specific pages and can be outside of the main website. 
Regarding documentation, Finding Get Started and Guide is straightfoward as in the navbar. 
It seems to be a free field, so I wonder how much should be added. We could also add "FAQ" doc, "reporting issue" documentation, etc... I was wondering what the scope was for this field.

Reply:
> can be outside of the main website.

So can the docs, see e.g. https://mamba.readthedocs.io/ 
I just followed an example I saw which had tutorial and documentation (don't remember where now though). I am indifferent about leaving it as is, omitting or adding something... Probably doesn't matter much.

Reply:
The scope for this field is to help users find information more quickly. Any sort of useful documentation can / should be put here. FAQ documents are probably the best example. If you run `winget show --id Microsoft.NuGet` you can see another example of where the CLI reference and Known Issues are specifically included.

I wouldn't go overboard on adding documentation if it is easy to find, but where it would be useful for users to be able to have a direct link to a page it is good to include.

Reply:
Thanks a lot for the explanation. We can add those url for now, and see what is the most useful in the future.

Reply:
Would you like me to change anything here @cderv? I am leaning towards FAQ and Documentation then, but like I said, not a strong preference.

Reply:
Oh, didn't see your reply yet, @cderv.